manifests/v/VMware/WorkstationPlayer/16.1.1/VMware.WorkstationPlayer.locale.en-US.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: VMware.WorkstationPlayer
+PackageVersion: 16.1.1
+PackageLocale: en-US
+Publisher: VMware, Inc.
+PackageName: VMware Player
+PackageUrl: https://www.vmware.com/products/workstation-player.html
+License: (c) 2021 VMware, Inc. All rights reserved.
+LicenseUrl: https://www.vmware.com/help/legal.html
+ShortDescription: VMware Workstation Player is a stable, mature solution for local desktop virtualization, allowing you to safely run a second operating system as a virtual machine on a single PC.
+Moniker: vmware-player
```

```suggestion
Moniker: vmware-workstation-player
```
Both applications have workstation in their name so